Cabal/Distribution/Compiler.hs

```diff
@@ -70,7 +69,7 @@ instance NFData CompilerFlavor where rnf = genericRnf
 
 knownCompilerFlavors :: [CompilerFlavor]
 knownCompilerFlavors =
-  [GHC, GHCJS, NHC, YHC, Hugs, HBC, Helium, JHC, UHC, Eta]
+  [GHC, GHCJS, NHC, YHC, Hugs, HBC, Helium, Eta]
```

Comment:
Given that we didn't remove Hugs, YHC or NHC from here, we should leave JHC and UHC here as well.

Reply:
Are you sure it wasn't just missed accidentally?

I understand that there might theoretically be Setup.hs code out there that relies on these, but wouldn't it be to "break it at compile time" rather than runtime?